v2/repository/github/queries_test.go

```diff
@@ -5,7 +5,7 @@ import (
 	"fmt"
 	"testing"
 
-	"github.com/golang/protobuf/protoc-gen-go/generator"
+	"github.com/iancoleman/strcase"
```

Comment:
why use this library instead of `protoc-gen-go/generator`?

Reply:
The `github.com/golang/protobuf` library is deprecated: https://github.com/golang/protobuf/blob/ae97035608a719c7a1c1c41bed0ae0744bdb0c6f/go.mod#L1

The suggested replacement to that library is https://github.com/protocolbuffers/protobuf-go , but it lacks the [`CamelCase`](https://github.com/golang/protobuf/blob/ae97035608a719c7a1c1c41bed0ae0744bdb0c6f/protoc-gen-go/generator/generator.go#L2648) conversion function that we're actually using: https://github.com/protocolbuffers/protobuf-go/search?q=CamelCase . It's spiritual successor is [`GoCamelCase`](https://github.com/protocolbuffers/protobuf-go/blob/fb30439f551a7e79e413e7b4f5f4dfb58e117d73/internal/strs/strings.go#L32), but that's in an internal package.

Iancoleman's library was just near the top of the Google hits for: "golang camelcase", and had an acceptable licence so I plugged it in.

Alternatively we could look at removing the camelcase step from the test? I didn't dig, just swapped one `ToCamelCase()` function for another.

Reply:
> Alternatively we could look at removing the camelcase step from the test? 

I took a look at this, and it works: https://github.com/grafeas/voucher/pull/39/commits/b24d50977ba27cb1072651e1120d5dceff82e277 🎉 
@dani-santos-code  thanks for flagging this smell!